tests/end-to-end/e2e/playground_test.go

```diff
@@ -9,80 +9,111 @@ import (
 	"io"
 	"log"
 	"net/http"
-	"os"
 	"testing"
 	"time"
 
-	retry "github.com/avast/retry-go"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/avast/retry-go"
 	"github.com/kyma-incubator/compass/components/director/pkg/graphql"
 	"github.com/kyma-incubator/compass/tests/end-to-end/pkg/connector"
 	"github.com/kyma-incubator/compass/tests/end-to-end/pkg/gql"
 	"github.com/kyma-incubator/compass/tests/end-to-end/pkg/idtokenprovider"
 	gcli "github.com/machinebox/graphql"
 	"github.com/stretchr/testify/require"
+	"github.com/vrischmann/envconfig"
 )
 
-func TestDirectorPlaygroundAccess(t *testing.T) {
-	const (
-		DirectorURLformat   = "https://%s.%s/director"
-		OAuth2Subdomain     = "compass-gateway-auth-oauth"
-		JWTSubdomain        = "compass-gateway"
-		ClientCertSubdomain = "compass-gateway-mtls"
-	)
+type playgroundTestConfig struct {
```

Comment:
Just a suggestion but I think it would be better to move all the boilerplate to the file with url builder and leave only Tests* here. At first look it's difficult to spot where actual tests start.